src/algebra/group/opposite.lean

```diff
@@ -98,14 +98,24 @@ We also generate additive structures on `αᵃᵒᵖ` using `to_additive`
 @[to_additive] instance [cancel_comm_monoid α] : cancel_comm_monoid αᵐᵒᵖ :=
 { .. mul_opposite.cancel_monoid α, .. mul_opposite.comm_monoid α }
 
-@[to_additive] instance [div_inv_monoid α] : div_inv_monoid αᵐᵒᵖ :=
+@[to_additive add_opposite.sub_neg_monoid] instance [div_inv_monoid α] : div_inv_monoid αᵐᵒᵖ :=
```

Comment:
Which name does `to_additive` autogenerate? If it's wrong here shouldn't it be corrected for the whole file (in particular by adding the rule to [`to_additive.tr`](https://github.com/leanprover-community/mathlib/blob/master/src/algebra/group/to_additive.lean#L202))?

Reply:
`add_opposite.sub_neg_add_monoid`. See [Zulip](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/naming.20contest/near/274315240).